### PR TITLE
build: set version to 4.0.0-beta.1

### PR DESCRIPTION
**Pull request by ckerr**
Bump the version to 4.0.0-beta.1

### CODE REVIEW

**Review by livings124 — APPROVED** [review]

**Review by mikedld — APPROVED** [review]

**Review by Coeur — APPROVED** [review]

File: news/news-4.0.0-beta-1.md [hunk 1]
```diff
@@ -26,39 +25,28 @@ Some of the highlights include:
...
-* Updated code to use modern JavaScript.
-* No longer use jQuery UI.
-* No longer use jQuery.
-* Use Webpack to bundle the Javascript, CSS, and assets together -- the entire bundle size is now 68K gzipped.
-* Added eslint / prettier / stylelint tooling.
-* Uses torrent-get's 'table' mode for more efficient RPC calls.
+- Support for using [BitTorrent v2](http://bittorrent.org/beps/bep_0052.html) torrents and [hybrid](http://bittorrent.org/beps/bep_0052.html#upgrade-path) torrents. (Support for _creating_ v2 and hybrid torrents is slated for an upcoming release.)
+- Users can now set "default" trackers that can be used to announce all public torrents.
+- Newly-added seeds can start immediately and verify pieces on demand, instead of needing a full verify before seeding can begin. ([#2626](https://github.com/transmission/transmission/pull/2626))
+- There now an option to omit potentially-identifying information (e.g. User-Agent and date created) when creating new torrents. ([#3452](https://github.com/transmission/transmission/pull/3452))
```
Comment (Coeur) [inline news/news-4.0.0-beta-1.md:44]: "There now" -> "There is now"

File: news/news-4.0.0-beta-1.md [hunk 2]
```diff
@@ -147,9 +135,9 @@ Maintainer highlights include:
...
..., depler, Dinesh Manajipet, Dmitry Antipov, Dmitry Serov, Dmytro Lytovchenko, dubhater, Dzmitry Neviadomski, Esa Varemo, evils, ewtoombs, FallenWarrior2k, Federico Bond, FluxState, Frank Aurich, FX Coudert, Gary Elshaw, goldsteinn, Greg Hazel, Guido Cella, Guido Vranken, Hakjoon Sim, Han Shen, Harm133, Ilkka, Ilkka Kallioniemi, IMurzich, Isabella Skořepová, Jelle van der Waa, Johan, Jonas Malaco, JP-Ellis, kakuhen, Kirill Ovchinnikov, Kobaxidze256, Koro, L2501, LaserEyess, Lucas Clemente Vella, lucaxvi, Luukas Pörtfors, luzpaz, Mark Deepwell, Markus Amalthea Magnuson, Matan Ziv-Av, Matt Joiner, maxz, Max Zettlmeißl, Michael Lopez, Michal Kubiak, mickaelifs, Mike Gelfand, Mike Gilbert, Mitchell Livingston, Mitch Livingston, Nathan Benichou, Nicholas Guriev, Noobsai, Norbert Papke, Oleg Chashko, orbital-mango, OscarCunningham, Pavel Borzenkov, Pedro Scarapicchia Junior, Peter Bailey, Petrprogs, Pierre Carru, qu1ck, razaq, RobCrowston, Robert Palmer, Robin Seth Ekman, Rosen Penev, Sam Marcus, Sander van Kasteel, Sergey Fedoseev, sewe2000, shelvacu, Stefan Talpalaru, SweetPPro, Tomáš Kelemen, Tyler, Viacheslav Chimishuk, Vik, Vincent Vinel, Vitaly Potyarkin, vuori, wiz78, and Xist12gh!
```
Comment (Coeur) [inline news/news-4.0.0-beta-1.md:138]: "A Cœur", "Antoine Cœur" and "Cœur" are all three me. Let's just keep "Cœur" if you don't mind.

File: news/news-4.0.0-beta-1.md (hunk 1 above)
Comment (ckerr) [reply news/news-4.0.0-beta-1.md]: Fixed in 32d02e5f9bd622fe9cfab8b7828390a496e5aa37

File: news/news-4.0.0-beta-1.md (hunk 2 above)
Comment (ckerr) [reply news/news-4.0.0-beta-1.md]: Fixed in 32d02e5f9bd622fe9cfab8b7828390a496e5aa37

:relieved: thanks for proofreading all of those!

**Comment by GaryElshaw** [comment]
Charles or Mike, i see the license for TC is about to expire?

[TeamCity Enterprise](https://www.jetbrains.com/teamcity/?fromServer) 2022.04.3 (build 108706) , open source license will expire in 2 days

**Pull request merged**